client/gutenberg/extensions/related-posts/view.js

```diff
@@ -0,0 +1,77 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { includes } from 'lodash';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import edit from './edit';
+import { ALIGNMENT_OPTIONS, MAX_POSTS_TO_SHOW } from './constants';
+
+registerBlockType( 'a8c/related-posts', {
```

Comment:
Aside, `registerBlockType` is one good argument for the file containing it to be understood as an entrypoint that should be named `index.js` :thinking: :roll_eyes:

Reply:
why are we registering a block for the editor in the view script anyway? shouldn't this belong in `editor.js`?

Reply:
https://github.com/Automattic/wp-calypso/pull/26530#issuecomment-422670783